kubejs/server_scripts/bedrock_fluid.js

```diff
@@ -1,37 +1,13 @@
-GTCEuServerEvents.fluidVeins(event => {
-
-    event.add('kubejs:void/oil', vein => {
-        vein.dimensions('javd:void')
-        vein.fluid(() => Fluid.of("gtceu:oil_medium").fluid)
-        vein.weight(1000)
-        vein.minimumYield(300)
-        vein.maximumYield(300)
-        vein.depletionAmount(0)
-        vein.depletionChance(0)
-        vein.depletedYield(300)
-    })
-
-// Luna fluid veins	
-	event.add('kubejs:moon/deuterium', vein => {
-        vein.dimensions('gcyr:luna')
-        vein.fluid(() => Fluid.of("gtceu:deuterium").fluid)
-        vein.weight(60)
-        vein.minimumYield(10)
-        vein.maximumYield(30)
-        vein.depletionAmount(1)
-        vein.depletionChance(50)
-        vein.depletedYield(5)
-    })
-	
-		event.add('kubejs:moon/helium_3', vein => {
-        vein.dimensions('gcyr:luna')
-        vein.fluid(() => Fluid.of("gtceu:helium_3").fluid)
-        vein.weight(35)
-        vein.minimumYield(5)
-        vein.maximumYield(20)
-        vein.depletionAmount(1)
-        vein.depletionChance(50)
-        vein.depletedYield(3)
-    })
-  
-})
\ No newline at end of file
+/*
+This file intentionally left blank as part of renaming or moving another file.
+
+This is done because pack update mechanisms naively assume files won't be deleted, renamed, or moved;
+They only alter files that are added or modified.
+
+As a result, if this file were not made blank then a player would update their pack to have both this old file
+AND the new file both exist at once with identical contents.
+
+This causes errors in KJS or other difficult-to-troubleshoot issues, 
+so we first update this old file to something nonfunctional as you see here
+and THEN delete it in the next patch after.
```

Comment:
@Xefyr0 Does that mean it would also work when someone skips the two updates? Otherwise, I'm afraid it wouldn't happen as rarely as we might think.

Reply:
No, it wouldn't.

It was an attempt to mitigate the problems that arose when people updated the modpack [as per this suggestion](https://discord.com/channels/914926812948234260/1297246927204778077/1297246927204778077), but I don't think I'm going to continue with the idea since:
1. It's a problem with launchers, and trying to fix it from our end will require a lot of extra work
2. Any effort we put in to fix it on our end won't matter if someone makes a mistake in one commit
3. CF launchers do not have this issue (Prism too, as of the 9.0 update)

So I'm thinking the best way forward is to forego this approach and encourage people to switch launchers if they have this issue in the future